Pinta.Resources/Pinta.Resources.csproj

```diff
@@ -15,7 +15,7 @@
     <PackageReference Include="GtkSharp" Version="3.24.24.34" />
   </ItemGroup>
   <ItemGroup>
-    <Content Include="icons/hicolor/16x16/*/*">
+    <Content Include="icons/hicolor/*/*/*">
```

Comment:
There probably also needs to be an update to `installer/linux/install.proj` to grab the svg icons during the `make install` step

Reply:
Good call, added!